### DISCUSSION TITLE
v1.8.1

### DISCUSSION THREAD

**Discussion by smira** [siderolabs/talos#9479] [Announcements]
## [Talos 1.8.1](https://github.com/siderolabs/talos/releases/tag/v1.8.1) (2024-10-08)

Welcome to the v1.8.1 release of Talos!

Starting with Talos v1.8.0, only standard assets would be published as github release assets. These include:

* `cloud-images.json`
* `talosctl` binaries
* `kernel`
* `initramfs`
* `metal` iso and disk images
* `talosctl-cni-bundle`

All other release assets can be downloaded from [Image Factory](https://www.talos.dev/latest/talos-guides/install/boot-assets/#image-factory).

Please try out the release binaries and report any issues at
https://github.com/siderolabs/talos/issues.

### Component Updates

Linux: 6.6.54
containerd: 2.0.0-rc.5
Flannel: 0.25.7

Talos is built with Go 1.22.8.

### Contributors

* Andrey Smirnov
* Hexoplon
* ekarlso

### Changes
<details><summary>16 commits</summary>
<p>

* siderolabs/talos@477752fe6 release(v1.8.1): prepare release
* siderolabs/talos@362c9f812 test: skip lvm test if not enough user disks available
* siderolabs/talos@79305007f chore: checkout extensions from release-1.8, not main
* siderolabs/talos@f6d630624 fix: wipe system partitions correctly via kernel args
* siderolabs/talos@4d279c65f fix: volume encryption with failing keyslots
* siderolabs/talos@070defad1 fix: update grpc-go the latest patch release
* siderolabs/talos@a2d12fd7b feat: update Flannel to v0.25.7
* siderolabs/talos@e2f560b96 feat: bring in lpfc kernel module driver
* siderolabs/talos@788336afb feat: enable QEDF driver
* siderolabs/talos@e4341fa66 fix: make /var/run empty on reboots
* siderolabs/talos@66228ef10 fix: multiple fixes for LVM activation
* siderolabs/talos@5f4515f30 fix: prevent file descriptors leaks to child processes
* siderolabs/talos@a55103ee6 chore: ignore more plugins for system containerd
* siderolabs/talos@ffcdc0bb7 fix: build talosctl without `tcell_minimal`
* siderolabs/talos@d29f66079 feat: add support for CI Network config in nocloud
* siderolabs/talos@01e580bdd feat: update Go 1.22.8, Linux, pkgs
</p>
</details>

### Changes from siderolabs/pkgs
<details><summary>8 commits</summary>
<p>

* siderolabs/pkgs@71d23b4 feat: update Linux to 6.6.54
* siderolabs/pkgs@8906a9b feat: add lpfc kernel module
* siderolabs/pkgs@3c57dff feat: enable QEDF driver
* siderolabs/pkgs@1ecbd58 feat: update containerd to v2.0.0-rc.5
* siderolabs/pkgs@47dff98 fix: drop the LVM2 udev lvm rule
* siderolabs/pkgs@480d765 fix: force LVM to use `/run` as state directory
* siderolabs/pkgs@c663212 feat: enable transparent huge pages in madvise mode
* siderolabs/pkgs@832f11b feat: update Go to 1.22.8
</p>
</details>

### Changes from siderolabs/tools
<details><summary>1 commit</summary>
<p>

* siderolabs/tools@7719230 feat: update Go to 1.22.8
</p>
</details>

### Dependency Changes

* **github.com/klauspost/compress**              v1.17.9 -> v1.17.10
* **github.com/siderolabs/go-blockdevice/v2**    v2.0.2 -> v2.0.3
* **github.com/siderolabs/pkgs**                 v1.8.0-8-gdf1a1a5 -> v1.8.0-16-g71d23b4
* **github.com/siderolabs/talos/pkg/machinery**  v1.8.0 -> v1.8.1
* **github.com/siderolabs/tools**                v1.8.0-1-ga0c06c6 -> v1.8.0-2-g7719230
* **google.golang.org/grpc**                     v1.66.0 -> v1.66.3

Previous release can be found at [v1.8.0](https://github.com/siderolabs/talos/releases/tag/v1.8.0)

## Images

```
ghcr.io/siderolabs/flannel:v0.25.7
registry.k8s.io/coredns/coredns:v1.11.3
gcr.io/etcd-development/etcd:v3.5.16
registry.k8s.io/kube-apiserver:v1.31.1
registry.k8s.io/kube-controller-manager:v1.31.1
registry.k8s.io/kube-scheduler:v1.31.1
registry.k8s.io/kube-proxy:v1.31.1
ghcr.io/siderolabs/kubelet:v1.31.1
ghcr.io/siderolabs/installer:v1.8.1
registry.k8s.io/pause:3.9
```

<hr /><em>This discussion was created from the release <a href='https://github.com/siderolabs/talos/releases/tag/v1.8.1'>v1.8.1</a>.</em>

**Comment by eshikhov** [comment]
Tested "66228ef10 fix: multiple fixes for LVM activation" in our cluster, LVM activation works. 

Thanks for the work you've done.
